### ISSUE TITLE
Entity Ids are not available in the docs.jsonld document

### ISSUE THREAD

**Issue by partageit** [api-platform/core#1201]
Hello,
Entity Ids are not available in my `docs.jsonld` document.

According to [this page](https://api-platform.com/docs/core/serialization-groups-and-relations#entity-identifier-case), adding a setter to the `id` field should be enough.
But it is not :-s

It seems to work with the [demo docs.jdsonld](https://demo.api-platform.com/docs.jsonld), where Ids are available.

The property annotations seems to be the same (except the `@groups`, but i have tried without), as well as the Id setter.

This is an extract of my Entity:

```php
/**
* @ORM\Entity
* @ApiResource(attributes={"filters"={"customer.search","customer.date"}})
*/
class Customer {
  /**
  * @ORM\Id
  * @ORM\Column(type="integer")
  * @ORM\GeneratedValue(strategy="AUTO")
  * @Groups({"mygroup"})
  */
  protected $id;

[...]
  public function getId() {
    return $this->id;
  }

  public function setId($id) {
    $this->id = $id;
    return $this;
  }
```

My versions: PHP 7.1 with API Platform 2.0.6.

After some investigations, in `DoctrineOrmPropertyMetadataFactory`:

    $writable = $doctrineClassMetadata->isIdentifierNatural();

Then `writable` is false, an identifier not writable is not returned in `docs.jsonld`.

It seems normal, but why is the behaviour of `demo.api-platform.com` different?

In other words, what did I miss?

**Comment by Simperfit** [comment]
Where does mygroup came from ? You should add it to the ApiResource if you use it.

Anyway this is [demo](https://github.com/api-platform/demo/blob/master/src/AppBundle/Entity/Book.php)'s code

**Comment by partageit** [comment]
The `myGroup` group comes from another entity, to nest items.

I tried with and without group, the result is the same.

I have already checked the demo's code, it seems to be very similar.

**Comment by partageit** [comment]
I have git-cloned the [demo repository](https://github.com/api-platform/demo) and I have the same result: `Book`.`id` and `Review`.`id` are missing from `docs.jsonld`.

These fields are present on your [demo platform `docs.jsonld`](https://demo.api-platform.com/docs.jsonld).

As sources are probably the same, what could be the difference?

I tried with many configurations:
- `php bin/console server:run`
- apache
- nginx + PHP FPM
- Symfony 3.3, Symfony 3.2

**Comment by dunglas** [comment]
The online demo currently use this branch: https://github.com/api-platform/demo/pull/15

Can you try with 2.1@beta?

**Comment by partageit** [comment]
You're right, it works with API Platform 2.1 :)
When will this version be available ?
Thank you for your answer, and for API Platform :)
